docs/topics_5/web-create-project.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='add_project_{context}']
+= Creating a project
+
+You can create a project in the {WebName} with the *Create project* wizard.
+
+.Procedure
+
+. In the {WebName}, click *Projects*.
+. Click *Create project*.
+. Enter a unique name for the project, an optional description, and click *Next*.
++
+. To upload applications, click the *Upload* tab, click *Browse*, select the application files you want to upload, and click *Close*.
++
+Uploading applications stores them directly on the {ProductShortName} server.
+
+. To register a server path, click the *Server path* tab and enter the *Server-side path* of the application in the field.
++
+Registering the server path of an application ensures that {ProductShortName} always uses the latest version.
+. Click *Next*.
+. Click one or more transformation targets.
++
+.Transformation targets
+image::web-configure-analysis_51.png[Transformation targets]
+. Click *Next*.
+. Select packages and click *>* to include them in the analysis.
+. Click *Next*.
++
+. If you want to add a custom rule, click *Add rule*.
```

Comment:
I just noticed this is an optional step. So in this case you can make the following two steps into bullet points under this step. It's not the most elegant way to do it but I think it's better than having upload/register path in the main procedure.